### ISSUE TITLE
TestInitialScalePostUpgrade failed as postupgrade test, after upgrading to 0.17 serving

### ISSUE THREAD

**Issue by houshengbo** [knative/operator#253]
https://prow.knative.dev/view/gs/knative-prow/pr-logs/pull/knative_operator/251/pull-knative-operator-upgrade-tests/1296540928600707074

Using operator to upgrade serving 0.16.0 to serving 0.17.0, TestInitialScalePostUpgrade tests failed after switching to 0.17.0 knative serving.

```
=== RUN   TestInitialScalePostUpgrade
=== PAUSE TestInitialScalePostUpgrade
=== CONT  TestInitialScalePostUpgrade
=== CONT  TestInitialScalePostUpgrade
    service_postupgrade_test.go:148: Getting service "init-scale-service"
=== CONT  TestInitialScalePostUpgrade
    service_postupgrade_test.go:154: Post upgrade Service is not ready with reason ""
--- FAIL: TestInitialScalePostUpgrade (0.20s)
```

**Comment by Cynocracy** [comment]
huh, interesting.

**Comment by Cynocracy** [comment]
In the logs I see
```
tive-tests.iam.gserviceaccount.com,},OwnerReferences:[]OwnerReference{},Finalizers:[],ClusterName:,ManagedFields:[]ManagedFieldsEntry{},} {{&ObjectMeta{Name:,GenerateName:,Namespace:,SelfLink:,UID:,ResourceVersion:,Generation:0,CreationTimestamp:0001-01-01 00:00:00 +0000 UTC,DeletionTimestamp:<nil>,DeletionGracePeriodSeconds:nil,Labels:map[string]string{},Annotations:map[string]string{autoscaling.knative.dev/window: 6s,},OwnerReferences:[]OwnerReference{},Finalizers:[],ClusterName:,ManagedFields:[]ManagedFieldsEntry{},}} {[{   <*>true <*>100 <nil>}]}} {{1 [{ConfigurationsReady False  {2020-08-20 20:35:02 +0000 UTC} RevisionFailed Revision "scale-to-zero-upgrade-service-lpdm8" failed with message: Initial scale was never achieved.} {Ready False  {2020-08-20 20:35:02 +0000 UTC} RevisionFailed Revision "scale-to-zero-upgrade-service-lpdm8" failed with message: Initial scale was never achieved.} {RoutesReady True  {2020-08-20 20:35:03 +0000 UTC}  }] <nil>} {scale-to-zero-upgrade-service-lpdm8 scale-to-zero-upgrade-service-lpdm8} {<*>http://scale-to-zero-upgrade-service.serving-tests.example.com <*>{<*>http://scale-to-zero-upgrade-service.serving-tests.svc.cluster.local} [{ scale-to-zero-upgrade-service-lpdm8  <*>true <*>100 <nil>}]}}} <nil>}
```

Maybe the cluster is getting close to full?

**Comment by tcnghia** [comment]
@vagababov is this same with the autoscaling bug?

**Comment by Cynocracy** [comment]
Is the bug that the autoscaler scales things down to 0 with an initScale?

**Comment by vagababov** [comment]
yeah it looks like the one we fixed already.
Will be released in 0.17.1.
